added note for pandas install, added section for data-analysis-projects (#39)
LaunchCodeEducation/data-analysis-curriculum: content/how-programs-work/reading/data-analysis-projects/index.md

+++
title = "GitHub Repository: Data Analysis Projects"
date = 2024-01-08T10:24:35-06:00
draft = false
weight = 7
+++

> Throughout this course you will be utilizing a GitHub repository that holds starter code for exercises, studios, and in some lessons, example code from readings.

Reviewer: ```suggestion
Throughout this course, you will be utilizing a GitHub repository that holds starter code for exercises, studios, and in some lessons, example code from readings.
```

Fork the following repository and clone it to your machine: [LaunchCode Education: Data Analysis Projects](https://github.com/launchcodeeducation/data-analysis-projects)

{{% notice blue Note "rocket" %}}

> You will begin using the `data-analysis-projects` repository beginning in the next chapter, do not leave here without it!

Reviewer: ```suggestion
You will begin using the `data-analysis-projects` repository beginning in the next chapter. Do not move on to the next page without it!
```
